flight/targets/coptercontrol/fw/pios_board.c

```diff
@@ -856,11 +856,13 @@ void PIOS_Board_Init(void) {
 #endif	/* PIOS_INCLUDE_PWM */
 		break;
 	case HWCOPTERCONTROL_RCVRPORT_PPM:
```

Comment:
Yah I didn't want to break existing stuff.  It's not the default PPM anymore which is a little weird, but... meh.